Resources/MessageQueue/MQConnectionManager.py

```diff
@@ -84,44 +85,47 @@ def addNewMessenger( self, mqURI, messengerType ):
     """
     # 'consumer1' ->1
     # 'producer21' ->21
-    msgIdToInt = lambda msgIds, msgType : [int(m.replace( msgType,'' )) for m in msgIds]
+    def msgIdToInt(msgIds, msgType):
+      return [int(m.replace(msgType, '')) for m in msgIds]
```

Comment:
Why do you changed? Because it is more pythonic?

Reply:
autopep8 changed it.